plugins/zapier/deploy_key.go

```diff
@@ -0,0 +1,75 @@
+package zapier
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/importer"
+	"github.com/1Password/shell-plugins/sdk/provision"
+	"github.com/1Password/shell-plugins/sdk/schema"
+	"github.com/1Password/shell-plugins/sdk/schema/credname"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+)
+
+func DeployKey() schema.CredentialType {
+	return schema.CredentialType{
+		Name:          credname.DeployKey,
+		DocsURL:       sdk.URL("https://platform.zapier.com/cli_docs/docs#quick-setup-guide"),
+		ManagementURL: sdk.URL("https://developer.zapier.com/partner-settings/deploy-keys/"),
+		Fields: []schema.CredentialField{
+			{
+				Name:                fieldname.Key,
+				MarkdownDescription: "Deploy Key used to authenticate to Zapier.",
+				Secret:              true,
+				Composition: &schema.ValueComposition{
+					Length: 32,
+					Charset: schema.Charset{
+						Lowercase: true,
+						Digits:    true,
+					},
+				},
+			},
+		},
+		DefaultProvisioner: provision.TempFile(
+			zapierConfig,
+			provision.AtFixedPath("~/.zapierrc"),
+		),
```

Comment:
Let's go with an envvar provisioner instead, as referenced here: https://platform.zapier.com/cli_docs/docs

ZAPIER_DEPLOY_KEY should do the trick. Reason is that the file provisioner currently has some limitations, so, wherever possible, I recommend using envvars.